eng/pipelines/maintenance-packages.yml

```diff
@@ -96,7 +96,46 @@ stages:
               $(_InternalBuildArgs)
             displayName: $(_BuildJobDisplayName)
 
+- stage: pack
+  displayName: Pack
+  jobs:
+  - template: /eng/common/templates/jobs/jobs.yml
+    parameters:
+      artifacts:
+        publish:
+          artifacts: false
+          logs: true
+          manifests: true
+      enableMicrobuild: true
+      enablePublishUsingPipelines: false
+      publishAssetsImmediately: false
+      enablePublishTestResults: false
```

Comment:
I set these to false because I expect that we only need the log output.
I want to see what happens when these are set to false and there's an error.